src/curves/ed25519.ts

```diff
@@ -15,6 +15,8 @@ export const isValidPrivateKey = (_privateKey: Uint8Array | string | bigint) =>
 
 export const deriveUnhardenedKeys = false;
 
+export const publicKeyLength = 33;
```

Comment:
These are 1 longer than I expected 🤔

Reply:
ED25519 public keys are prefixed with `0x00` for some reason.